README.md

```diff
@@ -9,29 +9,30 @@ This is my research and reflection journal for DGL 104 winter 2023
 - want to create some new things with learned technology or skill.
 - want to use this in an organization i am going to work and get better in using those new technology.
 - perfect in what i am learning in this course.
```

Comment:
The goals above haven't changed since I last looked at them, and so they are still missing some specifics and measures. However, you've made some good statements below that help.